src/js/components/FileInput/stories/Multiple.js

```diff
@@ -1,27 +1,41 @@
-import React from 'react';
+import React, { useState } from 'react';
```

Comment:
Since the "Disabled" story already captures this behavior, can we leave this story untouched?